yoda/llava_data_prep/scripts/edgar_ingestion.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+kit_dir = os.path.abspath(os.path.join(current_dir, "../.."))
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
+
+from yoda.llava_data_prep.src.edgar_ingestion import SECTools # type: ignore
+from yoda.llava_data_prep.src.table_utils import TableTools # type: ignore
+
+CONFIG_PATH = os.path.join(kit_dir, 'llava_data_prep', 'config.yaml')
+DATA_DIRECTORY = os.path.join(kit_dir, 'llava_data_prep', 'sec_data_yolo')
+
+sec_tool = SECTools(config=CONFIG_PATH)
+
+tickers = ["AAPL"]
+form_types = ["10-K"]
+after = "2023-01-01"
+before =  "2024-01-01"
+
+sec_tool.download_filings(tickers=tickers, 
```

Comment:
You may want to implement try-except blocks around the sec network calls and file operations to handle exceptions gracefully.